packages/client/src/App.tsx

```diff
@@ -1,21 +1,21 @@
-import { useCallback, useEffect, useState } from 'react'
+import React, { useCallback, useEffect, useState } from 'react'
 import { ThemeProvider } from '@emotion/react'
 import { theme } from './styles'
 import GlobalStyle from './styles/GlobalStyles'
 import { getAllInfo } from './apis/kiosk'
 import Tab from 'atoms/Tab'
 import styled from '@emotion/styled'
 import TabItem from 'atoms/Tab/TabItem'
-import React from 'react'
+import { TCategory } from 'types'
 
 const App = () => {
   // 데이터 필드 변경에 따른 ts타입 나중에 변경 예정
-  const [kioskData, setKioskData] = useState<any>()
+  const [kioskData, setKioskData] = useState<TCategory[] | undefined>(undefined)
```

Comment:
```suggestion
const [kioskData, setKioskData] = useState<TCategory[]>()
```

이 부분은 이렇게 해줘도 될 것 같은데 혹시 가독성을 위해 undefined를 적어주신 걸까요?

Reply:
타입스크립트의 타입 추론과 관련해서 다음 글을 읽어보셔도 좋을 것 같습니다!

https://www.typescriptlang.org/ko/docs/handbook/type-inference.html

Reply:
오 그러네요! 자료 잘 읽었습니다 감사합니다ㅎㅎ